.circleci/config.yml

```diff
@@ -17,32 +18,25 @@ commands:
           event: fail
           template: basic_fail_1
           branch_pattern: main
-jobs:
-  golang-lint:
-    executor:
-      name: go/default  # is based on cimg/go
-      tag: '1.21'
+  install-just:
+    description: "Install just"
     steps:
-      - checkout
-      - run:  golangci-lint run superchain/... validation/... add-chain/...
-  golang-modules-tidy:
+      - run: |
+          wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
+          echo "deb [arch=all,$(dpkg --print-architecture) signed-by=/usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg] https://proget.makedeb.org prebuilt-mpr $(lsb_release -cs)" | sudo tee /etc/apt/sources.list.d/prebuilt-mpr.list
+          sudo apt update
+          sudo apt install just
+
+jobs:
+  lint-and-tidy:
```

Comment:
We will need to sympathetically update the repo settings for "required" checks.

Reply:
In the end I reverted to the existing pattern, we can change this later if we want to.